src/themes/dspace/app/home-page/home-news/home-news.component.html

```diff
@@ -4,11 +4,10 @@
       <div class="d-flex flex-wrap">
         <div>
           <h1 class="display-3">DSpace 7</h1>
-          <p class="lead">DSpace is the world leading open source repository platform that enables
-            organisations to:</p>
+          <p class="lead">DSpace is a digital service that collects, preserves, and distributes digital material. Repositories are important tools for preserving an organization's legacy; they facilitate digital preservation and scholarly communication.</p>
```

Comment:
Please add this text into message in the en.json5 file for translating purposes.